transforms/helpers.ts

```diff
@@ -3,33 +3,55 @@ import {Collection, JSCodeshift, StringLiteral} from 'jscodeshift';
 export interface RemapJSXPropsConfig {
     [name: string]:
         | {
-        to: string;
-        values?: Record<any, string>;
-    }
+              to: string;
+              values?: Record<any, string>;
+          }
         | string;
 }
 
-export function remapJSXProps(name: string, config: RemapJSXPropsConfig, root: Collection, j: JSCodeshift) {
+export function remapJSXProps(
+    componentName: string,
+    config: RemapJSXPropsConfig,
+    root: Collection,
+    j: JSCodeshift,
+) {
+    const [mainName, childrenName] = componentName.split('.');
+
     const foundImport = root
         .find(j.ImportDeclaration, {
             source: {value: '@gravity-ui/uikit'},
             importKind: 'value',
         })
-        .find(j.ImportSpecifier, {type: 'ImportSpecifier', imported: {name}})
+        .find(j.ImportSpecifier, {type: 'ImportSpecifier', imported: {name: mainName}})
         .nodes()[0];
 
     if (!foundImport) {
         return false;
     }
 
-    root.find(j.JSXElement, {
-        openingElement: {
-            name: {
-                type: 'JSXIdentifier',
-                name: foundImport.local?.name || foundImport.imported.name,
-            },
-        },
-    })
+    root.find(
+        j.JSXElement,
+        childrenName
+            ? {
+                  openingElement: {
+                      name: {
+                          type: 'JSXMemberExpression',
+                          object: {
+                              type: 'JSXIdentifier',
+                              name: foundImport.local?.name || foundImport.imported.name,
```

Comment:
Why `childrenName` not used later? Looks like `Button.Something` would also match, but that's an error

Reply:
fixed